torch/ao/quantization/_dbr/mappings.py

```diff
@@ -110,6 +125,27 @@
     (torch.Tensor.mul, torch.Tensor.mul_),
 )
 
+conv_transpose_ops = set([
+    F.conv_transpose1d,
+    F.conv_transpose2d,
+    F.conv_transpose3d
+])
+
+conv_ops = set([
+    F.conv1d,
+    F.conv2d,
+    F.conv3d,
+]).union(conv_transpose_ops)
```

Comment:
nit: might be ambiguous to say that "conv transpose ops are conv ops", there are some arguments for and some against.  What do you think about keeping them separate, to make it more clear?

Reply:
Yes, I went back and forth on this too. For the most part both conv and conv transpose go through the same code paths, so there will be many places where we have to say `if op in conv_ops or op in conv_transpose_ops`, which is a bit verbose IMO. On the other hand, keeping them separate might be clearer as you suggested. I don't have a strong preference one way or the other.

Reply:
IMO usually the way which is clear and easy to read is preferrable in cases like this.